packages/itwin/grouping-mapping-widget/src/test/GroupingMappingCustomUI.test.tsx

```diff
@@ -186,8 +217,12 @@ describe("Groupings View with default UIs", () => {
       uiComponent: mockedUIComponent,
     };
 
+    groupingMappingCustomUIMock = [mockGroupingUI];
```

Comment:
Set to the default value (empty array) before each test.

Reply:
Done